Modules/Quotas/src/Quotas.Infrastructure.Database.SqlServer/Migrations/QuotasDbContextModelSnapshot.cs

```diff
@@ -43,6 +43,25 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("Identities");
                 });
 
+            modelBuilder.Entity("Backbone.Modules.Quotas.Domain.Aggregates.Identities.MetricStatus", b =>
+                {
+                    b.Property<string>("Owner")
+                        .HasColumnType("char(36)");
+
+                    b.Property<string>("MetricKey")
+                        .HasMaxLength(50)
+                        .IsUnicode(true)
+                        .HasColumnType("nvarchar(50)")
+                        .IsFixedLength(false);
+
+                    b.Property<DateTime?>("IsExhaustedUntil")
```

Comment:
You should re-generate the migrations, because `IsExhaustedUntil` is not nullable anymore. This was part of my changes. It is now set to `ExhaustionDate.Unexhausted`, which contains `DateTime.MinValue`